### DISCUSSION TITLE
Unable to display custom logo on the nav bar

### DISCUSSION THREAD

**Discussion by haris-bit** [squidfunk/mkdocs-material#5502] [Q&A] [answered]
I'm facing difficulties in displaying a custom logo on the nav bar of my MkDocs website using the Material theme. I have followed the official documentation and tried multiple approaches, but the logo is not appearing as expected. Here are the details of my setup:

MkDocs version: latest
Material theme version: 9.1.11
Operating system: Windows 11
Steps taken:
Created a custom folder in my project directory.
Placed my logo file (logo.png) inside the custom folder.
Updated the mkdocs.yml file with the necessary configurations to point to the logo file.
Expected behavior: The custom logo should be displayed on the nav bar of my website.
Actual behavior: The logo is not displaying, and instead, the alt text is shown as "logo".
I have tried different file paths, file extensions, and variations of the configurations, but none of them seem to work. I have also cleared the cache and reloaded the website, but the issue persists.

Could someone please help me troubleshoot this problem? Any guidance, suggestions, or examples of working configurations would be greatly appreciated. Thank you in advance for your assistance.

**Comment by kamilkrzyskow** [comment, marked as answer]
The actual configuration of both the `mkdocs.yml` and the file structure would help to pin point the problem.
Based on your message I envision a structure like this:
```
.
├── docs/
│   └── Some .md files
├── custom/
│   └── logo.png
└── mkdocs.yml
```
> Generated with https://tree.nathanfriend.io/

And inside your `mkdocs.yml`
```yml
theme:
  logo: custom/logo.png
```

This guide shows a path relative to the `docs` directory, or more specifically, to the root of your generated website, so adding the `custom` directory to the path is incorrect, since it's outside of the `docs` directory.
https://squidfunk.github.io/mkdocs-material/setup/changing-the-logo-and-icons/#image 

If I guessed correctly then change it to either one of those:
```
.
├── docs/
│   ├── assets/
│   │   └── images/
│   │       └── logo.png <-- Follow a certain structure when placing assets
│   └── Some .md files
└── mkdocs.yml   
------------------------
.
├── docs/
│   └── Some .md files
├── overrides/<-- My personal preference for the name
│   └── assets/
│       └── images/
│           └── logo.png <-- My personal preference to keep static assets in the overrides (works perfectly with MkDocs, but plugin developers tend to forget about this possibility. 
└── mkdocs.yml
```
and inside the configuration file:
```yml
theme:
    custom_dir: overrides # only needed for the second approach above
    logo: assets/images/logo.png
```
The path doesn't include the custom directory.

If I guessed incorrectly, then you could also run `mkdocs build` and see for yourself if the logo is put in the correct path.
You can also use the browsers developer tools to check the path given in the logo image. :v:

**Reply by haris-bit** [reply]
Thank you, It worked!
